Themes/default/Profile.template.php

```diff
@@ -1560,7 +1560,7 @@ function template_edit_options()
 					if (is_array($field['options']))
 						foreach ($field['options'] as $value => $name)
 							echo '
-							<option', is_numeric($value) ? ' value="" disabled' : ' value="' . $value . '"', $value === $field['value'] ? ' selected' : '', '>', $name, '</option>';
+							<option', (!empty($field['disabled']) && is_array($field['disabled']) && in_array($value, $field['disabled']) ? ' disabled' : ''), ' value="' . $value . '"', $value === $field['value'] ? ' selected' : '', '>', $name, '</option>';
```

Comment:
The basic idea here is good, but more work needs to be done before this could be merged without breaking existing functionality.

1. Please use `$field['disabled_options']` rather than `$field['disabled']`.
2. Add a line after [this one](https://github.com/SimpleMachines/SMF2.1/blob/release-2.1/Sources/Profile-Modify.php#L542) that sets 'disabled_options' to an array populated with any and all numeric keys returned from `smf_list_timezones()`.

Reply:
I can do those edits but want to ask a question prior to pushing the changed:
When viewing the page source of a disabled tag it just shows value ... it does not show value=""
Would you like that same behavior or not to bother omitting the value as I suggested?
Keep in mind that disabled elements are not submitted with the form (they are ignored).